hack/images

```diff
@@ -25,9 +25,9 @@ if [ -z "$TAG" ] || [ -z "$REPO" ]; then
   usage
 fi
 
-pushFlag="push=false"
+outputFlag="--output=type=docker"
```

Comment:
why docker?

Reply:
Because `make images` should load `moby/buildkit:local[-rootless]` to local Docker

Reply:
we might need to separate these cases then. as it is also used to test build in PRs where docker doesn't make sense.

Reply:
fixed